docker-storage-setup.sh

```diff
@@ -657,26 +657,36 @@ EOF
     Fatal "udevadm settle after partition creation failed. Exiting."
   fi
 
-  if ! wait_for_dev ${dev}1; then
-    Fatal "Partition device ${dev}1 is not available"
+  if ! wait_for_dev ${part}; then
+    Fatal "Partition device ${part} is not available"
```

Comment:
Creating partitions _is_ tricky, unfortunately.  See https://bugs.freedesktop.org/show_bug.cgi?id=85477, for example.

The explicit wait is probably not necessary, but even when using `lsblk` we can still wait until lsblk outputs the expected child.